lib/zendesk_apps_support/package.rb

```diff
@@ -94,7 +94,7 @@ def text_files
     end
 
     def js_files
-      @js_files ||= files.select { |f| f =~ %r{^(app|lib\/.*)\.js$} }
+      @js_files ||= files.select { |f| f =~ %r{^(.*)\.jsx?$} }
```

Comment:
Probably don't need the capturing group for `(.*)` here

Reply:
As discussed, I will be changing this to `%r{^.*\.jsx?$}`.